R/chrome.R

```diff
@@ -266,8 +279,45 @@ print_page = function(ws, url, output, wait, verbose, token, format, options = l
           ws$send('{"id":8,"method":"Runtime.evaluate","params":{"expression":"pagedownReady.then(() => {pagedownListener(\'\');})"}}')
       },
       # Command #8 received - No callback
-      NULL, {
-      # Command #9 received (printToPDF or captureScreenshot) -> callback: save to file & close Chrome
+      NULL,
+      # Command #9 received -> callback: command #10 DOM.getDocument
+      ws$send('{"id":10,"method":"DOM.getDocument"}'),
+      # Command #10 received -> callback: command #11 DOM.querySelector
+      ws$send(sprintf(
+        '{"id":11,"method":"DOM.querySelector","params":{"nodeId":%i,"selector":"%s"}}',
+        msg$result$root$nodeId,
+        selector
+      )), {
+      # Command 11 received -> callback: command #12 DOM.getBoxModel
+        if (msg$result$nodeId == 0)
+          token$error <- 'No element in the HTML page corresponds to the `selector` value.'
+        else
+          ws$send(sprintf('{"id":12,"method":"DOM.getBoxModel","params":{"nodeId":%i}}', msg$result$nodeId))
+      }, {
```

Comment:
I am not sure about the code style you want to apply. 
I find it more clear if we have explicit `{ }`, but I know R is ok with no `{`, and it is working when only one line
```r
if (cond) {
    do_something()
} else {
    do_other_thing()
}
```

I don't know if there is `{` missing here, or if I must adapt to your style for pagedown.  🤔 
I don't mind proof reading code, but need to know the rule 😊

Reply:
modified in https://github.com/rstudio/pagedown/pull/79/commits/d666dd33cf3eac365ae801e07020fc9e723e2f61